libraries-5/src/test/java/com/baeldung/githubapi/RepositoryUnitTest.java

```diff
@@ -0,0 +1,95 @@
+package com.baeldung.githubapi;
+
+import com.google.common.base.Charsets;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.Test;
+import org.kohsuke.github.*;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class RepositoryUnitTest {
```

Comment:
If this is making a live connection it should be a `LiveTest` so it doesn't make the CI build flaky on connection issues